### ISSUE TITLE
A question regarding the mutual information part of the loss function

### ISSUE THREAD

**Issue by dandansheep0** [uta-smile/TVT#19]
Hi, author.  I have a question regarding the mutual information part of the loss function. In the main file, I noticed that this part of the loss is written as:
if args.use_im:
    loss += (args.theta * loss_im)

However, your paper mentions that this part should be maximized. I’m wondering why it’s not written as loss -= (args.theta * loss_im)? I also didn’t see any negation operation for this part in lossZoo.im(). 

I’m not entirely sure if my understanding is correct, so I would appreciate it if you could help clarify this for me.

Thank you!

**Comment by viyjy** [comment]
can you check equation 9 and it's explanation? I remember that loss_im is already negative
